server_admin/topics/authentication/webauthn.adoc

```diff
@@ -12,22 +12,17 @@ NOTE: Whether WebAuthn's operations succeed depends on a user's WebAuthn support
 
 The setup procedure of WebAuthn support for 2FA is the following :
 
-===== Enable User Registration
-
-An administrator carries out the following operations on the `Admin Console` :
-
-- Open the `Realm Settings -> Login` tab.
-- Set the `User Registration` to ON and click `Save`.
-
+[[_webauthn-register]]
 ===== Enable Webauthn Authenticator Registration
 
 An administrator carries out the following operations on the `Admin Console` :
 
 - Open the `Authentication -> Required Actions` tab.
 - Click `Register`.
 - Select `Webauthn Register` as `Required Action`.
-- Mark `Enabled` and `Default Action` checkbox.
+- Mark `Enabled` checkbox. Optionally mark `Default Action` checkbox if you want all new created users to be required to register WebAuthn credential.
 
+[[_webauthn-authenticator-setup]]
 ===== Adding WebAuthn Authentication to a Browser Flow
 
 * Select a realm, click on Authentication link, select the "Browser" flow
```

Comment:
The Authentication word above should be probably rendered with background too, IOW change it to _click on the `Authentication` link, select the "Browser flow"_

Currently the sentence is rendered like this:
![authentication](https://user-images.githubusercontent.com/8414918/72801193-43714f80-3c49-11ea-9a4b-cfe3f8a948b5.png)

Reply:
Will fix it, Thanks